python_modules/dagster/dagster/_core/definitions/auto_materialize_rule.py

```diff
@@ -368,6 +382,35 @@ def materialize_on_required_for_freshness() -> "MaterializeOnRequiredForFreshnes
         """
         return MaterializeOnRequiredForFreshnessRule()
 
+    @public
+    @staticmethod
+    def materialize_on_cron(
+        cron_schedule: str, timezone: str = "UTC", all_partitions: bool = False
+    ) -> "MaterializeOnCronRule":
+        """Materialize an asset partition if it has not been materialized since the previous cron
```

Comment:
```suggestion
        """Materialize an asset partition if it has not been materialized since the latest cron
```

I see wht you mean here by "previous", but I think "latest" or "most recent" would be less ambiguous.